src/components/@shared/AssetList/index.tsx

```diff
@@ -1,19 +1,7 @@
 import AssetTeaser from '@shared/AssetTeaser'
-import React, { ReactElement, useEffect, useState } from 'react'
+import React, { ReactElement } from 'react'
 import Pagination from '@shared/Pagination'
 import styles from './index.module.css'
-import Loader from '@shared/atoms/Loader'
-import { useIsMounted } from '@hooks/useIsMounted'
-import { getAccessDetailsForAssets } from '@utils/accessDetailsAndPricing'
```

Comment:
so this method `getAccessDetailsForAssets()` should in the end disappear completely, but seeing we are using it for bookmarks and stats on profile page. Can we also get rid of this? Or do in new PR?